client/posts/get-the-first-scrollable-parent-of-an-element/index.tsx

```diff
@@ -5,36 +5,38 @@ import RelatedPosts from '../../components/RelatedPosts';
 
 export default () => {
     return (
-<>
-<Markdown
-    content={`
-Starting from the given \`ele\` element, we traverse all parents up to the root of document (\`document.body\`).
-For each parent node, we check if it is a [scrollable node](/check-if-an-element-is-scrollable).
+        <>
```

Comment:
Thanks @elkarouani.
Can you remove all the indentation?

Reply:
Ok, I will work on it